profiles/views_test.py

```diff
@@ -35,16 +42,16 @@
 from search.base import ESTestCase
 
 
-class ProfileTests(ESTestCase):
+class ProfileBaseTests(ESTestCase):
     """
-    Tests for GET on profile view
+    Tests for profile views
     """
 
     def setUp(self):
         """
         Create a user and profile
         """
-        super(ProfileTests, self).setUp()
+        super(ProfileBaseTests, self).setUp()
 
         with mute_signals(post_save):
```

Comment:
just curious: shouldn't the creation of `user1` and `user2` etc live inside of `setUpTestData` rather than `setUp`?